### PR TITLE
editHtml

### PR DESCRIPTION
**Pull request by EstebanIndiveri**
#### Summary

Added newArticle Screen Editor HTML

changed confirmation screen
added LeftHeader Confirmation Screen
Added helperStlye navigation
Added a navUtil Reset

### Ios
Captura de Pantalla 2021-09-23 a la(s) 12 41 12
Captura de Pantalla 2021-09-23 a la(s) 12 44 22
Simulator Screen Recording - iPhone 12 - 2021-09-23 at 12 45 43 (1)

### Android

Captura de Pantalla 2021-09-23 a la(s) 12 27 01
Captura de Pantalla 2021-09-23 a la(s) 12 27 44

### CODE REVIEW

File: src/app/components/CrossBack/index.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import { NavigationProp } from '@react-navigation/core';
+import { onResetStack } from '@utils/navUtils';
+import { useDispatch } from 'react-redux';
```
Comment (andreszuluaga-wolox) [inline src/app/components/CrossBack/index.tsx:5]: organize

File: src/app/screens/NewArticle/index.tsx [hunk 2]
```diff
@@ -106,105 +119,115 @@ function NewArticle({ route: { params } }: Props) {
       );
     }
   };
-
+  const handleContentSizeChange = () => {
```
Comment (andreszuluaga-wolox) [inline src/app/screens/NewArticle/index.tsx:122]: try ternary operator `const handleContentSizeChange = () => isIos && scrollViewRef?.current?.scrollToPosition(0, height - height / 1.2, true); `

File: __tests__/src/constants/navigationHelper.test.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { customStyles } from '../../../src/constants/navigationHelper';
```
[the remark this answers is not in the document]
Comment (EstebanIndiveri) [reply __tests__/src/constants/navigationHelper.test.ts]: fixed

File: src/app/screens/NewArticle/index.tsx (hunk 2 above)
Comment (EstebanIndiveri) [reply src/app/screens/NewArticle/index.tsx]: refactored

**Review by andreszuluaga-wolox — APPROVED** [review]

File: src/app/components/CrossBack/index.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import actionCreators from '@redux/articles/actions';
+import { onResetStack } from '@utils/navUtils';
+
+import styles from './styles';
+
+type Props = {
+  navigation: NavigationProp<any>;
+};
+
+export default function CrossBack({ navigation }: Props) {
```
Comment (john07289) [inline src/app/components/CrossBack/index.tsx:14]: why no use useNavigationWithParams?

Comment (EstebanIndiveri) [reply src/app/components/CrossBack/index.tsx]: changed!

**Review by john07289 — APPROVED** [review]

**Pull request merged**